src/feature-flags.js

```diff
@@ -1,16 +1,32 @@
-const featureFlags = [
-  {
-    name: 'userProfile',
-    isActive: true,
-  },
-  {
-    name: 'applicationTracker',
-    isActive: false,
-  },
-  {
-    name: 'profileSupportServices',
-    isActive: false,
-  },
-];
+const featureFlags = {
```

Comment:
We can actually condense the checking of the env by throwing it in here! example:

```
const envConfigs = {...}
const featureFlags = envConfigs[process.env]
export default featureFlags
```

Reply:
looks nice! will do.